README.md

```diff
@@ -1,5 +1,17 @@
 ## Consul on ECS
 
+<p>
+  <a href="https://www.consul.io/docs/ecs/#gh-light-mode-only" target="_blank" title="Consul ECS documentation">
```

Comment:
> `/#gh-light-mode-only`

The downside of this is it takes us to `https://www.consul.io/docs/ecs#gh-dark-mode-only` with the trailing selector